test/jdk/java/awt/print/PageFormat/Orient.java

```diff
@@ -21,64 +21,42 @@
  * questions.
  */
 
-/*
-  @test
-  @bug 4236095
-  @summary  Confirm that the you get three pages of output, one
-            each in portrait, landscape, and reverse landscape
-            orientations.
-  @key printer
-  @run main/manual Orient
-*/
-
-
-//*** global search and replace Orient with name of the test ***
+import java.awt.Color;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.geom.Ellipse2D;
+import java.awt.print.Book;
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterException;
+import java.awt.print.PrinterJob;
 
-/**
- * Orient.java
- *
- * summary:
+/*
+ * @test
+ * @bug 4236095
+ * @summary  Confirm that you get three pages of output, one
+ *           each in portrait, landscape, and reverse landscape
+ *           orientations.
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @key printer
+ * @run main/manual Orient
  */
-
-import java.awt.*;
-import java.awt.event.*;
-import java.awt.geom.*;
-import java.awt.print.*;
-
-// This test is a "main" test as applets would need Runtime permission
-// "queuePrintJob".
-
 public class Orient implements Printable {
-
-   private static void init()
-    {
-        //*** Create instructions for the user here ***
-
-        String[] instructions =
-        {
-         "On-screen inspection is not possible for this printing-specific",
-         "test therefore its only output is three printed pages.",
-         "To be able to run this test it is required to have a default",
-         "printer configured in your user environment.",
-         "",
-         "Visual inspection of the printed page is needed. A passing",
-         "test will print three pages each containing a large oval ",
-         "with the text describing the orientation: PORTRAIT, LANDSCAPE",
-         "or REVERSE_LANDSCAPE, inside of it. The first page will ",
-         "be emitted in portait orientation, the second page in landscape ",
-         "orientation and the third page in reverse-landscape orientation. ",
-         "On each page the oval will be wholly within the imageable area ",
-         "of the page. In a failing test the oval on the third page ",
-         "will be clipped against the imageable area.",
-         "Axes will indicate the direction of increasing X and Y"
-       };
-      Sysout.createDialog( );
-      Sysout.printInstructions( instructions );
-
+    private static final String INSTRUCTIONS =
+            "This test will automatically initiate a print.\n\n" +
```

Comment:
Can use Text Block style for better readability.

String INSTRUCTIONS = """
...""".